wrappers/javascript/aries-askar-shared/src/store/StoreKeyMethod.ts

```diff
@@ -0,0 +1,18 @@
+export enum KdfMethod {
+  Raw = 'raw',
+  None = 'none',
+  Argon2IMod = 'kdf:argon2i:mod',
+  Argon2IInt = 'kdf:argon2i:int',
+}
+
+export class StoreKeyMethod {
+  private method: KdfMethod
+
+  public constructor(method: KdfMethod) {
+    this.method = method
+  }
+
+  public toUri() {
+    return this.method.toString()
+  }
+}
```

Comment:
Is this for future proofing? We could just use the enum value directly right?!

Reply:
Yes, it is for future proofing. Argon could have a salt or other kdf methods might need some custom logic.